src/lib/CurrencyInput.svelte

```diff
@@ -78,12 +78,20 @@
 
 	// Formats the value when the input loses focus and sets the correct number of
 	// fraction digits when the value is zero
-	const handleOnBlur = () => setFormattedValue(true);
+	const handleOnBlur = () => setFormattedValue();
 
-	// Also set the correct fraction digits when the value is zero on initial load
-	onMount(() => setFormattedValue(true));
+	let dom: Document;
+	let inputElement: HTMLInputElement;
 
-	let inputTarget: HTMLInputElement;
+	onMount(() => {
+		// Set the document object as a variable so we know the page has mounted
+		dom = document;
+
+		// Set the correct fraction digits when the value is zero on initial load
+		setFormattedValue()
+	});
+
+	let inputTarget: EventTarget | null;
```

Comment:
Used to be:
`let inputTarget: HTMLInputElement;`

But the correct type should have been:
`let inputTarget: EventTarget | null;`